processor/stream/package_tests/span_attrs_test.go

```diff
@@ -41,7 +41,7 @@ func spanProcSetup() *tests.ProcessorSetup {
 
 func spanPayloadAttrsNotInFields() *tests.Set {
 	return tests.NewSet(
-		"span.stacktrace",
+		tests.Group("span.stacktrace"),
```

Comment:
why is this changed?

Reply:
This is due to the change in tests.flattenMapStrStr, to iterate over `[]common.MapStr` fields. Previously there would be an entry for "span.stacktrace", but not for any of the fields in the objects within it.